index.js

```diff
@@ -35,18 +40,18 @@ function organizeMoonsByPlanet(moons) {
 
   You should not need to make any other changes to the code.
 */
-function getClosestToPlanet(moons) {
-  let closest = moons[0];
-  const rest = moons.slice(1);
+const getClosestToPlanet = (moons) => {
+
+  let [closest, ...rest] = moons;
 
   for (let moon of rest) {
-    if (moon.distanceFromPlanetInKm < closest.distanceFromPlanetInKm) {
+    if (moon.distanceFromPlanetInKm < closest.distanceFromPlanetInKm)
```

Comment:
I'm not going to make an argument for it in this space, but you'll find that for _many_ good reasons, professional teams _always_ write `if` statements with braces , even for one-statement blocks. Skipping the braces is a very good way to introduce a huge and hard-to-pin-down bug, and the savings in time and space is minimal.

Don't get yourself into any bad habits over the next year!

Reply:
Thank you.